Source/WPEFramework/Config.h

```diff
@@ -477,6 +483,9 @@ namespace PluginHost {
             Observables Observe;
 #ifdef HIBERNATE_SUPPORT_ENABLED
             HibernateConfig Hibernate;
+#endif
+#ifndef __DISABLE_USE_COMPLEMENTARY_CODE_SET__
```

Comment:
Have you verified by enabling the __DISABLE_USE_COMPLEMENTARY_CODE_SET__ switch?